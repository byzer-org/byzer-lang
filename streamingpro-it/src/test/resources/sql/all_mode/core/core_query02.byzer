load parquet.`/ssb/customer/part-00000-7505a64d-b4bf-4186-874a-a9898502a51e-c000.snappy.parquet` as customer_parquet;

select * from customer_parquet as customer;

select * from customer where C_CUSTKEY > 202 and C_CUSTKEY < 206 as customer_filter;
