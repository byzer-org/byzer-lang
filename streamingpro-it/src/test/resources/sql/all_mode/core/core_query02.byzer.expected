C_CUSTKEY,C_NAME,C_ADDRESS,C_CITY,C_NATION,C_REGION,C_PHONE,C_MKTSEGMENT
203,Customer#000000203,ZL9FxQJrF4aIDo,ARGENTINA8,ARGENTINA,AMERICA,11-563-614-7615,MACHINERY
204,Customer#000000204,U6TxYh,FRANCE   7,FRANCE,EUROPE,16-837-482-3566,BUILDING
205,Customer#000000205,Ic7aplPrDJMpexX1fi,JAPAN    2,JAPAN,ASIA,22-437-131-9769,BUILDING